spec.html

```diff
@@ -25857,7 +25800,7 @@ <h1>Object.getOwnPropertySymbols ( _O_ )</h1>
 
         <emu-clause id="sec-getownpropertykeys" aoid="GetOwnPropertyKeys">
           <h1>Runtime Semantics: GetOwnPropertyKeys ( _O_, _type_ )</h1>
-          <p>The abstract operation GetOwnPropertyKeys is called with arguments _O_ and _type_ where _O_ is an Object and _type_ is one of the ECMAScript specification types String or Symbol. The following steps are taken:</p>
+          <p>The abstract operation GetOwnPropertyKeys takes arguments _O_ and _type_ (either String or Symbol). It performs the following steps:</p>
```

Comment:
Elsewhere we replaced the union of String and Symbol with property key. Should we do that everywhere?

Reply:
Note that this isn't _a_ String or Symbol. It's literally String or Symbol. This parameter takes the _name_ of a type. (Open to suggestions for how to word this better; I guess I could put it back closer to what it was before.)

Reply:
Oh, should it be `~string~ or ~symbol~`?

Reply:
Then the implementation would have to change. Currently this parameter is used as `If Type(_nextKey_) is _type_`.

Reply:
Easy enough: https://github.com/tc39/ecma262/pull/1918